netlib/http_semantics.py

```diff
@@ -0,0 +1,23 @@
+class Response(object):
```

Comment:
it was already a class somewhere - and somewhere else it was used as namedtuple...
I decided a proper class is probably cleaner.